saleor/graphql/product/tests/test_product_pagination.py

```diff
@@ -466,6 +467,114 @@ def test_products_pagination_with_sorting_by_attribute(
     assert len(products_nodes) == page_size
 
 
+def test_products_pagination_for_products_with_the_same_names_two_pages(
```

Comment:
What do you thing to test product slugs in response, to check ordering by slug? In this case we should have `slug-1` and `slug-2` in first page and `slug-3` in second page, right?

Reply:
Yes, I can check it too.